README.md

```diff
@@ -10,23 +8,29 @@
 [![codecov](https://codecov.io/gh/codecov/example-typescript/branch/master/graph/badge.svg)](https://codecov.io/gh/codecov/example-typescript)
 [![Build Status](https://travis-ci.org/codecov/example-typescript.svg?branch=master)](https://travis-ci.org/codecov/example-typescript)
 
-Run npm install:
-```shell
-npm install
+## Guide
+### Travis Setup
+Add to your `.travis.yml` file.
+```yml
+language: java
```

Comment:
Don't think the language should be `java`

Reply:
Ah yes. Thanks for the catch!